### PR TITLE
Feature/componentpreviewtransforms

### PR DESCRIPTION
**Pull request by dwilliamson67**

### CODE REVIEW

File: src/Components/ComponentPreviewTransformModel.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        /// </summary>
+        public long ComponentPreviewTransformsId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the component preview transform.
+        /// </summary>
+        public string ComponentPreviewTransformName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identity of the component type.
```
Comment (deevezina) [inline src/Components/ComponentPreviewTransformModel.cs:45]: Fix comment

File: src/Components/ComponentPreviewTransformModel.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        /// </summary>
+        public DateTime CreateDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identity of the component type.
+        /// </summary>
+        public long ComponentTypeId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the review state information.
```
Comment (deevezina) [inline src/Components/ComponentPreviewTransformModel.cs:55]: Fix comment

File: src/Components/ComponentPreviewTransformModel.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        /// </summary>
+        public ComponentProcessType State { get; set; } = ComponentProcessType.Translation;
+
+        /// <summary>
```
Comment (deevezina) [inline src/Components/ComponentPreviewTransformModel.cs:59]: What's this?

**Review by deevezina — CHANGES_REQUESTED** [review]

File: src/Components/ComponentPreviewTransformModel.cs (hunk 1 above)
Comment (dwilliamson67) [reply src/Components/ComponentPreviewTransformModel.cs]: done

File: src/Components/ComponentPreviewTransformModel.cs (hunk 2 above)
Comment (dwilliamson67) [reply src/Components/ComponentPreviewTransformModel.cs]: done

File: src/Components/ComponentPreviewTransformModel.cs (hunk 3 above)
Comment (dwilliamson67) [reply src/Components/ComponentPreviewTransformModel.cs]: Must have been on crack at the time. removed

**Review by deevezina — APPROVED** [review]

**Pull request merged**